tests/x509/test_x509.py

```diff
@@ -263,6 +266,7 @@ def test_get_revoked_certificate_by_serial_number(self, backend):
         )
         serial_number = 725064303890588110203033396814564464046290047507
         revoked = crl.get_revoked_certificate_by_serial_number(serial_number)
+        assert isinstance(revoked, x509.RevokedCertificate)
```

Comment:
Why is this required?

Reply:
`get_revoked_certificate_by_serial_number ` returns `Optional[RevokedCertificate]` so we need to assert on it. We can do isinstance or we could just assert not None, but otherwise the next line will fail